network/modules/UDP/socket_wrapper.py

```diff
@@ -0,0 +1,102 @@
+import socket
+
+
+class UdpSocket:
+    """
+    Wrapper for Python's socket module.
+    """
+
+    def __init__(self, socket_instance: socket.socket) -> None:
```

Comment:
I think you need to give it a default value of None for it to allow not passing in any arguments